machine/dell/dell_s4000_c2338/installer.conf

```diff
@@ -0,0 +1,51 @@
+# DELL S4000 Series
+# CPU Module: Intel Rangeley (C2338)
+
+description="DELL S4000 Series w/ Intel Atom Rangeley (C2338) CPU Module"
+
+# Default ONIE block device
+install_device_platform()
+{
+    # The problem we are trying to solve is:
+    #
+    #    How to determine the block device upon which to install ONIE?
+    #
+    # The question is complicated when multiple block devices are
+    # present, i.e. perhaps the system has two hard drives installed
+    # or maybe a USB memory stick is currently installed.  For example
+    # the mSATA device usually shows up as /dev/sda under Linux, but
+    # maybe with a USB drive connected the internal disk now shows as
+    # /dev/sdb.
+    #
+    # The approach here is to look for the block device that has a
+    # very specific, bus centric path in the system.  The internal
+    # CFast device can only show up with this address.
+
+    # Find the internal CFast device -- this path is specific to this platform
+    #dell_s4000_c2338_disk1 option below is for units having 2 CFs.
+    #dell_s4000_c2338_disk2 is for unit having one CF.
+    dell_s4000_c2338_disk1="/sys/devices/pci0000:00/0000:00:17.0/host0/target0:0:0/0:0:0:0"
+    dell_s4000_c2338_disk2="/sys/devices/pci0000:00/0000:00:18.0/host4/target4:0:0/4:0:0:0"
+    for d in /sys/block/* ; do
+        if [ -e "$d/device" ] ; then
+            path="$(realpath $d/device)"
+            if [ "$path" = "$dell_s4000_c2338_disk1" ] ; then
+                device="/dev/$(basename $d)"
+                echo $device
+                return 0
+            elif [ "$path" = "$dell_s4000_c2338_disk2" ] ; then
+                device="/dev/$(basename $d)"
+                echo $device
+                return 0
+            fi
+        fi
+    done
```

Comment:
Does this mean you plan to ship two different SKUs for the S4000?

Will the EEPROM indicate the different SKUs, i.e. a different model number?